Makefile

```
@@ -441,6 +441,42 @@ acc_balance_query_app1: ## Query the balance of app1
 acc_balance_total_supply: ## Query the total supply of the network
 	poktrolld --home=$(POKTROLLD_HOME) q bank total --node $(POCKET_NODE)
 
+##############
+### Claims ###
+##############
+
+.PHONY: claim_create
+claim_create: ## Create a dummy claim by supplier1
+	poktrolld --home=$(POKTROLLD_HOME) tx supplier create-claim \
+	7b226170706c69636174696f6e5f61646472657373223a22706f6b74316d7271743566377168387578733237636a6d397437763965373461397676646e71356a766134222c2273657276696365223a7b226964223a22616e76696c222c226e616d65223a22227d2c2273657373696f6e5f73746172745f626c6f636b5f686569676874223a2231222c2273657373696f6e5f6964223a2273657373696f6e5f6964222c2273657373696f6e5f656e645f626c6f636b5f686569676874223a2235227d \
+	726f6f745f68617368 \
+	--from supplier1 --node $(POCKET_NODE)
+
+.PHONY: claims_list
+claim_list: ## List all the claims
+	poktrolld --home=$(POKTROLLD_HOME) q supplier list-claims --node $(POCKET_NODE)
+
+.PHONY: claims_list_address
+claim_list_address: ## List all the claims for a specific address (specified via ADDR variable)
+	poktrolld --home=$(POKTROLLD_HOME) q supplier list-claims --supplier-address $(ADDR) --node $(POCKET_NODE)
+
+.PHONY: claims_list_address_supplier1
+claim_list_address_supplier1: ## List all the claims for supplier1
+	SUPPLIER1=$$(make poktrolld_addr ACC_NAME=supplier1) && \
+	ADDR=$$SUPPLIER1 make claim_list_address
```

```suggestion
	ADDR=$$(make poktrolld_addr ACC_NAME=supplier1) make claim_list_address
```
can we just do this?

---

@h5law I had a similar initial reaction; it may seem more verbose in this specific example but after looking at several of the make targets which follow this structure, I found that it reads nicer holistically.